ext/suse/puppet.spec

```diff
@@ -9,8 +9,8 @@ Release: 1%{?dist}
 License: Apache 2.0
 Group:    Productivity/Networking/System
 
-URL: http://puppetlabs.com/projects/puppet/
-Source0: http://puppetlabs.com/downloads/puppet/%{name}-%{version}.tar.gz
+URL: https://puppetlabs.com/projects/puppet/
```

Comment:
That link still works (via a redirect), and is under Marketing's control. Which I think is as it should be? I'd leave it as is.